### PR TITLE
Implement loading changes into the image

### PR DESCRIPTION
**Pull request by marcelgarus**
Merge #87 first

### CODE REVIEW

File: packages/ChangesReloaded-Core.package/CRBrowser.class/instance/loadSelectedChanges.st [hunk 1]
```diff
@@ -0,0 +1,5 @@
+button-actions
+loadSelectedChanges
+	
+	self selectedItems do: [ :each |
+		each changeRecord fileIn ]
```
Comment (SilvanVerhoeven) [inline packages/ChangesReloaded-Core.package/CRBrowser.class/instance/loadSelectedChanges.st:5]:
```suggestion
		each fileIn ]
```

File: packages/ChangesReloaded-Core.package/CRBrowser.class/instance/loadSelectedChanges.st [hunk 2]
```diff
@@ -0,0 +1,5 @@
+button-actions
+loadSelectedChanges
+	
+	self selectedItems do: [ :each |
```
Comment (SilvanVerhoeven) [inline packages/ChangesReloaded-Core.package/CRBrowser.class/instance/loadSelectedChanges.st:4]:
```suggestion
	self selectedItems do: [:each |
```

File: packages/ChangesReloaded-Core.package/CRBrowser.class/instance/loadSelectedChanges.st (hunk 1 above)
Comment (SilvanVerhoeven) [inline packages/ChangesReloaded-Core.package/CRBrowser.class/instance/loadSelectedChanges.st:5]:
```suggestion
		each changeRecord fileIn]
```

File: packages/ChangesReloaded-Core.package/CRBrowser.class/instance/selectedItems.st [hunk 3]
```diff
@@ -0,0 +1,4 @@
+accessing
+selectedItems
+
+	^ self changeListItems select: [ :each | each isSelected ]
```
Comment (SilvanVerhoeven) [inline packages/ChangesReloaded-Core.package/CRBrowser.class/instance/selectedItems.st:4]:
```suggestion
	^ self changeListItems select: [:each | each isSelected]
```

File: packages/ChangesReloaded-Core.package/CRBrowser.class/instance/loadSelectedChanges.st (hunk 1 above)
Comment (strawberriesandcheese) [reply packages/ChangesReloaded-Core.package/CRBrowser.class/instance/loadSelectedChanges.st:5]: wait why?

File: packages/ChangesReloaded-Core.package/CRBrowser.class/instance/loadSelectedChanges.st [hunk 4]
```diff
@@ -0,0 +1,5 @@
+button-actions
+loadSelectedChanges
+	
+	self selectedItems do: [:each |
+		each changeRecord fileIn ]
```
Comment (strawberriesandcheese) [inline packages/ChangesReloaded-Core.package/CRBrowser.class/instance/loadSelectedChanges.st:5]:
```suggestion
		each fileIn]
```

File: packages/ChangesReloaded-Core.package/CRBrowser.class/instance/loadSelectedChanges.st [hunk 5]
```diff
@@ -0,0 +1,5 @@
+button-actions
+loadSelectedChanges
+	
+	self selectedItems do: [:each |
+		each fileIn]
```
Comment (strawberriesandcheese) [inline packages/ChangesReloaded-Core.package/CRBrowser.class/instance/loadSelectedChanges.st:5]:
```suggestion
	self selectedItems do: [:each | each fileIn]
```

**Comment by marcelgarus** [comment]
Ooops, I didn't think this through. 😬
I thought we could remove the `changeRecord`, because we implement the `noSuchMethodError:` to delegate methods to the `changeRecord`. But this only works as soon as the other PR is merged – since this is an independent PR for now, we should probably revert my last change.
We can (and should) still update the code as soon as the other PR is merged, but that's probably not the only place where we could do things more elegantly, so I suggest we do that at a later point in time.

**Comment by strawberriesandcheese** [comment]
ok #87 **_needs_** to be merged first but it still doesn't fail because we don't test it. this is fine

File: .smalltalk.ston [hunk 6]
```diff
@@ -6,11 +6,5 @@ SmalltalkCISpec {
       #load : 'Tests',
       #platforms : [ #squeak ]
     }
-  ],
-  #testing : {
-    #coverage : {
-      #packages : [ 'ChangesReloaded-Core.*' ]
-    },
-    #packages : [ 'ChangesReloaded-Tests.*' ]
-  }
```
Comment (strawberriesandcheese) [inline .smalltalk.ston:15]: This change will break our coverage reporting. Was this intentional?

File: .travis.yml [hunk 7]
```diff
@@ -1,5 +0,0 @@
-language: smalltalk
-sudo: false
-
-smalltalk:
-  - Squeak-trunk
```
Comment (strawberriesandcheese) [inline .travis.yml:5]: We need this for coverage reporting. (See above)

File: README.md [hunk 8]
```diff
@@ -1,4 +1,4 @@
-# ChangesReloaded ![Build Status](https://github.com/hpi-swa-teaching/ChangesBrowser/workflows/CI/badge.svg?branch=dev)[![Coverage Status](https://coveralls.io/repos/github/hpi-swa-teaching/ChangesBrowser/badge.svg?branch=dev)](https://coveralls.io/github/hpi-swa-teaching/ChangesBrowser?branch=dev)
+# ChangesReloaded [![Build Status](https://travis-ci.org/HPI-SWA-Teaching/ChangesBrowser.svg?branch=team09)](https://travis-ci.org/HPI-SWA-Teaching/ChangesBrowser)
```
Comment (strawberriesandcheese) [inline README.md:1]: rip coverage

File: .smalltalk.ston (hunk 6 above)
Comment (SilvanVerhoeven) [reply .smalltalk.ston:15]: nope

File: README.md (hunk 8 above)
Comment (SilvanVerhoeven) [reply README.md:1]: same same but different

File: .travis.yml (hunk 7 above)
Comment (SilvanVerhoeven) [reply .travis.yml:5]: what you guys need... ;D

**Pull request merged**